lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2234,8 +2200,8 @@ private static function get_block_nodes( $theme_json, $selectors = array() ) {
 				'name'       => $name,
 				'path'       => array( 'styles', 'blocks', $name ),
 				'selector'   => $selector,
+				'selectors'  => $feature_selectors,
 				'duotone'    => $duotone_selector,
```

Comment:
With `'selectors'` added is there some way we can deprecate the `'selector'` and `'duotone'` keys so they aren't relied upon in the future? It's part of the public API, so it probably can't be removed, but maybe updating the docblock could be enough?

Reply:
I'd like to handle this refactor in a follow-up, hopefully keeping the scope of changes here a little more manageable and easier to review.